src/export.js

```diff
@@ -1,18 +1,19 @@
-// Deprecated
-// Extend assert methods to QUnit for Backwards compatibility
-( function() {
-	var i,
-		assertions = Assert.prototype;
+function applyDeprecated( name ) {
+	return function() {
+		throw new Error(
+			"The global `" + name + "` is now removed from QUnit 2.0, you should " +
+			"now use the correct namespace. Check out our upgrade guide at " +
+			"https://qunitjs.com/upgrade-guide-2.x/"
+		);
+	};
+}
 
-	function applyCurrent( current ) {
-		return function() {
-			var assert = new Assert( QUnit.config.current );
-			current.apply( assert, arguments );
-		};
-	}
+( function() {
+	var i;
+	var assertions = Assert.prototype;
```

Comment:
Okay. I personally prefer 1 `var` per line, but everywhere else in QUnit we use a single `var`.